packages/data-store/src/api/user.js

```diff
@@ -55,5 +55,9 @@ export default {
 
   },
   
+  logout: function({ url }) {
+    url = url || `${this.apiUrl}/auth/project/${projectId}/logout?useAuth=oidc`;
+    document.location.href = url;
```

Comment:
Kan deze functie ook de session/localstorage variabelen weghalen, zoals de CMSuser en openstaduser etc? Zou denk ik ideaal zijn om dit niet zelf te moeten doen per plek dat je logout aanroept

Reply:
Die users gaan in principe vanzelf goed: je kunt nog wel iets in je storage hebben maar na het uitlogen is dat geen geldige data en doet die niets meer.

Maar het is denk ik wel een goed idee om alle sessiondata weg te gooien bij het uitloggen; je kunt volhouden dat een sessie dan eindigt. Ga ik toevoegen.

Reply:
Het zit nu in de currentUser hook